src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_ILocalFunctionStatement.cs

```diff
@@ -570,6 +570,135 @@ public void M(int x)
             VerifyOperationTreeAndDiagnosticsForTest<ArrowExpressionClauseSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TestLocalFunction_StaticWithShadowedVariableReference()
+        {
+            string source =
+@"#pragma warning disable 8321
+class C
+{
+    static void M(int x)
+    {
+        /*<bind>*/
+        static int Local(int y)
+        {
+            if (y > 0)
+            {
+                int x = (int)y;
+                return x;
+            }
+            return x;
+        }
+        /*</bind>*/
+    }
+}";
+            string expectedOperationTree = @"
+ILocalFunctionOperation (Symbol: System.Int32 Local(System.Int32 y)) (OperationKind.LocalFunction, Type: null) (Syntax: 'static int  ... }')
```

Comment:
>ILocalFunctionOperation [](start = 0, length = 23)

Can you add a work item in test plan (or a discrete issue) to track exposing `IsStatic` on the `ILocalFunctionOperation` node?

Reply:
As we discussed offline, callers can use `ILocalFunctionOperation.Symbol.IsStatic` instead.

---
In reply to: [245145913](https://github.com/dotnet/roslyn/pull/32067#discussion_r245145913) [](ancestors = 245145913)